public/app/controllers/userController.js

```diff
@@ -60,7 +60,7 @@ userControllers.controller('userEditController', function($http, User, $routePar
   });
 
   vm.updatedUser = {
-    username: ""
+    username: "",
```

Comment:
Hmm, yes I think so too. I was fiddling with that and just forgot to remove the comma.